.github/workflows/verify-opensource-rpm-installable.yml

```diff
@@ -24,7 +24,7 @@ jobs:
         run: |
           dnf list llvm-libs --showduplicates
           dnf install -y dnf-plugins-core
-          dnf config-manager --add-repo https://copr.fedorainfracloud.org/coprs/g/vespa/vespa/repo/epel-8/group_vespa-vespa-epel-8.repo
+          dnf -y config-manager --add-repo https://raw.githubusercontent.com/vespa-engine/vespa/master/dist/vespa-engine.repo
```

Comment:
I guess we should update documentation and all places where that repository is mentioned?

I found some:
- Vespa.AI Org: https://github.com/search?q=org%3Avespaai%20copr.fedorainfracloud.org&type=code
- Vespa Engine Org: https://github.com/search?q=org%3Avespa-engine%20copr.fedorainfracloud.org&type=code

Reply:
In general yes. But old blogs and references to CentOS 7 should be left alone.